### ISSUE TITLE
Providing option to restrict element-picking to a particular type

### ISSUE THREAD

**Issue by nzfeng** [nmwsharp/polyscope#92]
This came up when I was preparing the coding assignments for CMU's DDG course.

Currently, the element picker picks up on elements of all sorts, including vertices, edges, faces, halfedges. On the other hand, in some assignments, students only need to select vertices on a mesh. Vertex selection is still doable, but on larger meshes it can require zooming in to more precisely select vertices.

It seems we've discussed this problem before (just logging it here.) One thing I'll add is that it might be helpful if the user could restrict to a certain element type by holding down a key while clicking.

**Comment by nmwsharp** [comment]
Totally agree with this one! 

I've thought about it also that it's kinda crazy that you can pick "halfedges" by default, even those most users might not even know what a "halfedge" means, much less want to click on one. I was thinking one nice default would be to only allow picking faces & vertices by default, and then enable to other elements only once you've registered some data on them.

And also as you say, it would be nice to allow manually enabling various elements for picking as needed. Hopefully all of these can be added at the same time.

**Comment by nmwsharp** [comment]
Is this a blocker for the course? Or just something that would be nice to have?

**Comment by nzfeng** [comment]
> Is this a blocker for the course? Or just something that would be nice to have?

Nope, just something that would be nice to have!

**Comment by trilbytim** [comment]
Plus one for this feature! For what I'm working on, I'm only interested in picking faces. I have if statements at the moment to ignore it when something that's not a face is selected, but that means you have to keep clicking around to get it to find a face.

**Comment by min-hieu** [comment]
I wonder if there is any update on this?
